### ISSUE TITLE
Rotate conquest boards

### ISSUE THREAD

**Issue by nitaku** [notapixelstudio/superstarfighter#599]
Boards are always visible, so the game is scaled a lot and things are barely discernible. If boards have a more fitting aspect ratio, this should be alleviated.
